src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs

```diff
@@ -381,5 +381,10 @@ private void OnFilterMouseLeftButtonUp(object sender, MouseButtonEventArgs e)
         {
             FilterPopup.IsOpen = true;
         }
+
+        private void OnViewLayoutSelectorMouseLeftButtonUp(object sender, MouseButtonEventArgs e)
+        {
+            LayoutPopup.IsOpen = true;
```

Comment:
Do we really need null check? `LayoutPopup` is created in xaml and it will never be null.